ztp/siteconfig-generator/main.go

```diff
@@ -11,13 +11,17 @@ import (
 )
 
 func main() {
+	localExtraManifestPath := flag.String("manifestPath", "", "Directory where pre-defined extra manifest exist")
```

Comment:
Minor language fix-up suggestion:
```suggestion
	localExtraManifestPath := flag.String("manifestPath", "", "Directory with pre-defined extra manifests")
```

Reply:
done